peekingduck/pipeline/nodes/model/yolo.py

```diff
@@ -23,7 +23,72 @@
 
 class Node(AbstractNode):
     """Yolo node class that initialises and use yolo model to infer bboxes
-    from image frame
+    from image frame.
+
+    The yolo node is capable of detecting objects from 80 categories. It uses
+    YOLOv4-tiny by default and can be changed to using YOLOv4.
+
+    Inputs:
+        |img|
+
+    Outputs:
+        |bboxes|
+
+        |bbox_labels|
+
+        |bbox_scores|
+
+    Configs:
+        model_type (:obj:`str`): **{"v3", "v3tiny", "v4", "v4tiny"}, default="v4tiny"**
+
+            defines the type of YOLO model to be used.
+
+        weights_dir (:obj:`List`):
+            directory pointing to the model weights.
+
+        blob_file (:obj:`str`):
+            name of file to be downloaded, if weights are not found in `weights_dir`.
+
+        graph_files (:obj:`Dict`):
+            dictionary pointing to path of the model weights file.
+
+        size (:obj:`int`): **default = 416 **
+
+            image resolution passed to the YOLO model.
+
+        num_classes (:obj:`int`): **default = 80 **
+
+            maximum number of objects to be detected.
+
+        detect_ids (:obj:`int`): **default = 0 **
+
+            list of object class ids to be detected.
```

Comment:
is this a list or an int?

Reply:
Sorry, overlooked. This is a list of ids.